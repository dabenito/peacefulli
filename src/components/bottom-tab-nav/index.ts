export { Nav } from "./bottom-tab-nav";
