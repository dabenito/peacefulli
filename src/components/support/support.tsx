import React, { PureComponent } from "react";
import { View, Text } from "react-native";
import { styles } from "./style";

export interface SupportProps {}

export interface SupportState {}

export class Support extends PureComponent<SupportProps, SupportState> {
  constructor(props: SupportProps) {
    super(props);
  }

  render() {
    return (
      <View style={styles.container}>
        <Text>TODO: Create Support View!</Text>
      </View>
    );
  }
}
