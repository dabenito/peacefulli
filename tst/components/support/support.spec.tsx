import React from "react";
import { render, cleanup } from "@testing-library/react-native";
import { Support } from "../../../src/components/support";

describe("Support Tests", () => {
  afterEach(cleanup);

  // TODO: update this file when implementing support page
  it("loading support page", async () => {
    const support = render(<Support />);
    expect(support).toMatchSnapshot();
  });
});
