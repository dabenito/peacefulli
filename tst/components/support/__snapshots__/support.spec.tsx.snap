// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Support Tests loading support page 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    TODO: Create Support View!
  </Text>
</View>
`;
